source/gems/dry-validation/basics/macros.html.md

```diff
@@ -9,7 +9,9 @@ This document describes available built-in macros.
 
 ### filled
 
-Use it when a value is expected to be filled.
+Use it when a value is expected to be filled. "filled" means that the value is
+non-nil, and if the value is a `String`, `Hash`, or `Array` then `value.empty?`
+returns `false`.
```

Comment:
Well [in that case...](https://github.com/dry-rb/dry-rb.org/pull/107#discussion_r85852803) ;)